ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -340,61 +326,103 @@ Item {
+            anchors.topMargin: Style.current.padding
+            spacing: Style.current.bigPadding
+
+            Loader {
+                id: emptyViewAndSuggestionsLoader
+                active: communityData.amISectionAdmin &&
+                        (!localAccountSensitiveSettings.hiddenCommunityWelcomeBanners ||
+                        !localAccountSensitiveSettings.hiddenCommunityWelcomeBanners.includes(communityData.id))
+                width: parent.width
+                height: {
+                    // I dont know why, the binding doesn't work well if this isn't here
+                    item && item.height
+                    return this.active ? item.height : 0
+                }
+                sourceComponent: Component {
+                    CommunityWelcomeBannerPanel {
+                        activeCommunity: communityData
+                        store: root.store
+                        hasAddedContacts: root.hasAddedContacts
+                        communitySectionModule: root.communitySectionModule
+                        onManageCommunityClicked: root.manageButtonClicked()
+                    }
                 }
             }
-        }
 
-        Loader {
-            id: backUpBannerLoader
-            active: communityData.amISectionAdmin &&
-                        (!localAccountSensitiveSettings.hiddenCommunityBackUpBanners ||
-                         !localAccountSensitiveSettings.hiddenCommunityBackUpBanners.includes(communityData.id))
-            width: parent.width
-            height: active ? item.height : 0
-            anchors.top: emptyViewAndSuggestionsLoader.bottom
-            anchors.topMargin: active ? Style.current.padding : 0
-            sourceComponent: Component {
-                Item {
-                    width: parent.width
-                    height: backupBanner.height
-
-                    BackUpCommuntyBannerPanel {
-                        id: backupBanner
-                        activeCommunity: communityData
-                        onBackupButtonClicked: {
-                            Global.openPopup(transferOwnershipPopup, {
-                                privateKey: communitySectionModule.exportCommunity(communityData.id),
-                                store: root.store
-                            })
+            Loader {
+                id: channelsAndCategoriesAdminBox
+                active: communityData.amISectionAdmin &&
+                        (!localAccountSensitiveSettings.hiddenCommunityChannelAndCategoriesBanners ||
+                        !localAccountSensitiveSettings.hiddenCommunityChannelAndCategoriesBanners.includes(communityData.id))
+                width: parent.width
+                height: {
+                    // I dont know why, the binding doesn't work well if this isn't here
+                    item && item.height
+                    return this.active ? item.height : 0
```

Comment:
Strange, in theory, the binding should bind to `item` also.
How about making it more explicit, not sure if in Qt6, these `item && item.height`, at first glance unused, entries won't be optimized out:
```suggestion
                    return this.active && item && item.height  ? item.height : 0
```

Reply:
Have no idea also, the docs say it should work: `"Behind the scenes, the QML engine monitors the property's dependencies (that is, the variables in the binding expression). When a change is detected, the QML engine re-evaluates the binding expression and applies the new result to the property."`